lib/OpenQA/Worker/Job.pm

```diff
@@ -1080,12 +1079,8 @@ sub _upload_log_file {
         my $tx = $ua->build_tx(POST => $ua_url => form => $upload_parameter);
 
         if ($regular_upload_failed) {
-            log_warning(
-                sprintf(
-                    'Upload attempts remaining: %s/%s for %s, in %s seconds',
-                    $retry_counter--, $retry_limit, $filename, $tics
-                ));
-            OpenQA::Utils::wait_with_progress($tics);
+            log_warning(sprintf('Upload attempts remaining: %s/%s for %s', $retry_counter--, $retry_limit, $filename));
+            sleep 5;
         }
```

Comment:
Sorry, but I don't want to accept a sleep with magic number

Reply:
Please be more specific, we do not have a standard for this.

Reply:
E.g. use the constant `UPLOAD_DELAY` as I suggested in my review comment.

Reply:
I don't like constants for this, but ok.

Reply:
The constant `UPLOAD_DELAY` is defined in line 49 of the same file

```perl
use constant UPLOAD_DELAY        => $ENV{OPENQA_UPLOAD_DELAY} // 5;
```

I agree with @Martchus that you can just use the same constant so just

```perl
sleep UPLOAD_DELAY;
```

With `UPLOAD_DELAY` being configurable by environment variable you can even simplify the test mocking by just setting the variable `$ENV{OPENQA_UPLOAD_DELAY} = 0;` :)

Reply:
Making something configurable only at compile time is still rather clunky.

Reply:
Resolved.